### ISSUE TITLE
Object types fail to parse if the first element does not have a type annotation

### ISSUE THREAD

**Issue by sandersn** [Microsoft/TypeScript#10545]
If elements are separated by comma. Parsing succeeds if elements are separated by semi-colons.

```ts
let x: { foo: number, bar };
let y: { foo, bar };
let z: { foo, bar: number };
```

Then:
```ts
x: { foo: number, bar: any }
```

**Expected behavior:**

```ts
y: { foo: any, bar: any }
z: { foo: any, bar: number }
```
**Actual behavior:** 
`y` and `z` do not parse correctly.

**Comment by sandersn** [comment]
In `parsePropertyOrMethodSignature`, it looks like it should work &mdash; the type annotation appears to be optional. I'd have to step through the example to see why it fails.

**Comment by sandersn** [comment]
The actual parsing is fine, it was the `isTypeMemberStart` predicate that didn't know about commas. Fix is up at #10568
